lab-carolina/lib/read.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = function(file){
+  var array = [];
+  fs.readFile('text1.txt', function(err, data){
+    if(err) throw err;
+    array.push(data.toString('hex', 0, 8));
+    fs.readFile('text2.txt', function(err, data){
+      if(err) throw err;
+      array.push(data.toString('hex', 0, 8));
+      fs.readFile('text3.txt', function(err, data){
+        if(err) throw err;
+        array.push(data.toString('hex', 0, 8));
+        file(array);
```

Comment:
The name of this callback param threw me off for a second. It really should be `cb` or `callback` or something along those lines.